src/View.elm

```diff
@@ -1,31 +1,31 @@
 module View exposing (view)
 
-import Date
-import Date.Format exposing (format)
+import Time exposing (Time)
 import Html exposing (..)
 import Html.App
 import Html.Attributes exposing (id, for, attribute, class, type', value)
+import Utils exposing (timeAgo)
 import Model exposing (Model, Record, Msg(..))
 import Form
 
 
-formatLatsModified : Float -> String
-formatLatsModified timestamp =
-    format "%Y-%m-%d@%H:%M" (Date.fromTime timestamp)
+formatLastModified : Int -> Time -> String
+formatLastModified timestamp currentTime =
+    timeAgo (toFloat timestamp) currentTime
```

Comment:
I don't understand the `toFloat` here, is it to convert the `Int` to something that is "compatible" with a `Time`? From https://github.com/n1k0/kinto-elm-experiments/pull/2/files#diff-05d2065a68c705b72271bf365cdc74eeR6 i thought `timeAgo` was taking a Time, not a float?!